synapse/config/server.py

```diff
@@ -1257,7 +1257,9 @@ def add_arguments(parser: argparse.ArgumentParser) -> None:
             help="Turn on the twisted telnet manhole service on the given port.",
         )
 
-    def read_gc_intervals(self, durations) -> Optional[Tuple[float, float, float]]:
+    def read_gc_intervals(
+        self, durations: Optional[List[Any]]
```

Comment:
I don't think we know that this is a `str | int` at this point? It is just passed whatever is in the YAML object so it can be a dict, list, None, int, string, boolean.

Reply:
Ah yes indeed. Though following this logic, how do we know that it's a list? afaict we feed it the config directly without checking the type.

Reply:
Good point! I'll change it to `Any`.